.github/workflows/linux-eic-shell.yml

```diff
@@ -304,24 +304,37 @@ jobs:
         name: codecov_report
         path: build/codecov_report/
 
+  detector-info:
```

Comment:
Would you mind tucking this into some other job, e.g. `env`. Extra jobs require time overhead for startup, and take space in the listing.

Reply:
We can fold `env` into this, but `env` right now takes 0 time since it's trivial... cvmfs is the expensive part at about 30 second. So, don't expect any speedup by joining them. In fact, it will cause `build` to wait 30 seconds longer before it start running. Overall, I think the better way is to keep them separate, allow `build` to start promptly, and to allow `npsim` to start as soon as `detector-info` finishes.

Reply:
Okay, let's keep it mind.

Reply:
Alternative is to copy-paste this bit around